tests/baselines/reference/arrowFunctionWithObjectLiteralBody1.js

```diff
@@ -2,4 +2,4 @@
 var v = a => <any>{}
 
 //// [arrowFunctionWithObjectLiteralBody1.js]
-var v = function (a) { return {}; };
+var v = function (a) { return ({}); };

```

Comment:
Looks like a regression; is this known? We'd need to file a bug after this PR goes in.